gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/store/ServerGroupUtils.java

```diff
@@ -110,7 +110,7 @@ public static String getMyGroups() {
   /** return true if this VM is member of one of the given groups */
   public static boolean isGroupMember(Set<String> groups) {
     if (groups == null || groups.size() == 0) {
-      return true;
+      return false;
```

Comment:
This is not correct and will break things left and right. An empty/null group is used to denote the default group of the distributed system which includes all the members which is why this returns true. Can you explain how this "fixes" SNAP-1199? We need to find a proper fix.